modules/extensions/src/brushing/brushing-extension.ts

```diff
@@ -101,4 +101,17 @@ export default class BrushingExtension extends LayerExtension {
       });
     }
   }
+
+  draw(this: Layer<BrushingExtensionProps>, params: any, extension: this) {
+    const {viewport, mousePosition} = params.context;
+    const {brushingEnabled, brushingRadius, brushingTarget} = params.moduleParameters;
+    const brushingProps: BrushingModuleSettings = {
```

Comment:
Nit: variable name should follow type. We already have some confusion between "props" and "settings", this makes it worse
```suggestion
// either
    const brushingProps: BrushingModuleProps = {
// or
    const brushingSettings: BrushingModuleSettings = {
```